types/express-boom/index.d.ts

```diff
@@ -0,0 +1,353 @@
+// Type definitions for express-boom
```

Comment:
If this is for an npm package, please add a [corresponding version](https://gist.github.com/sandeepsuvit/2486c99c0346db87de24539472f34451).